tests/filecheck/projects/riscv-backend-paper/add.mlir

```diff
@@ -42,4 +42,4 @@ builtin.module {
     }
 }
 
-// CHECK: [[1.0, 2.25, 3.5], [4.75, 6.0, 7.25]]
+// CHECK: [1.0, 2.25, 3.5], [4.75, 6.0, 7.25]
```

Comment:
hmm

Reply:
ok maybe CHECK-SAME is the way here also, but the outer [] is actually significant

Reply:
I'll give it another try then!